### DISCUSSION TITLE
Click Event

### DISCUSSION THREAD

**Discussion by wadehats** [jeffreylanters/react-unity-webgl#466] [Help Wanted!] [answered]
Hey, all I am using this plug-in to run a program and I need to run something when a specific button has been clicked any ideas or suggestions?

**Comment by jeffreylanters** [comment, marked as answer]
Hi! A button in your Unity app, or in your React app?

**Reply by wadehats** [reply]
yes, is there a way to send an event from the unity app to the browser?

**Reply by jeffreylanters** [reply]
Yes, there is. You can read more about it in [the documentation](https://react-unity-webgl.dev/docs/api/event-system). If you're having any questions, feel free to ask them.
